pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -43,6 +43,10 @@ const (
 	pluginNotImplementingScore        = "plugin-not-implementing-score"
 	preFilterPluginName               = "prefilter-plugin"
 	preFilterWithExtensionsPluginName = "prefilter-with-extensions-plugin"
+	successFilterPlugin               = "success-filter-plugin"
```

Comment:
I don't think you need different names for the tests, and even if you do, I think you can just directly put it in your test case. No need to define a constant.